lib/utils/get-note.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import filterNotes from './filter-notes';
+
+export default function getNote( state ) {
+	if ( state.note ) {
+		return state.note;
+	}
+	const filteredNotes = filterNotes( state );
+	const noteIndex = Math.max( state.previousIndex, 0 );
+	return filteredNotes[ noteIndex ];
+}
```

Comment:
how about `getActiveNote()`?

it's just that I would expect `getNote()` to take some kind of id parameter and return a specific note